compare high and low res data

Tinkaa opened the pull request:
Notebooks to compare the differences between the high and low resolution data

I did some file renaming of notebooks so now the diff seems a lot larger than it is... Sorry for that.. 
Maybe the easiest would be to just run the notebooks and see if it makes any sense? 
Also not all conclusions are included in the notebooks as they are spread across presentations. Oh messymess

turnerm commented on analyses/mwi/dryspells_trigger/exploration/ecmwf_cds_compare_aggr.md:
````diff
@@ -177,18 +179,19 @@ def combine_for_obs(df_for,df_obs):
 ```python
 for key, values in meth_dict.items():
-    values["df"]=load_forecast_data(country_iso3,config,values["use_unrounded_area_coords"],resolution=values["resolution"],all_touched=values["all_touched"],
+    values["df"]=load_forecast_data(country_iso3,config,values["use_unrounded_area_coords"],
````
I think this needs to be updated to reflect the new filenames (e.g. with the coordinate bounds)

Tinkaa commented:
@turnerm I changed the code such that the area coords are read. Now everything should work! 

I did it in an ugly way though by hardcoding the area coords string. Let me know if you think this goes a step too far and should be done neater. 

Previously we didn't round the area coords while downloading the data (which is done in `pa-aa-toolbox`). Now that we do, I reran all the notebooks and updated the figures in the slides [here](https://docs.google.com/presentation/d/1jBdZEnzWwOKoFA9-h4xR4x0FmTyMRDCNBkWGKptfZs8/edit#slide=id.ge75e9e42c8_2_20)

turnerm commented on analyses/mwi/dryspells_trigger/09_ecmwf_monthly_skill_dryspells.md:
```diff
@@ -258,7 +259,9 @@ end_date="2-1-2020"
 # but results actually worsen with this
 date_list=pd.date_range(start=f'1-1-{start_year}', end=end_date, freq='MS')
-all_files=[processing.get_stats_filepath(country_iso3,config,date,resolution=resolution,adm_level=1,use_unrounded_area_coords=use_unrounded_area_coords,all_touched=all_touched) for date in date_list]
+all_files=[processing.get_stats_filepath(country_iso3,config,date,resolution=resolution,
```
I'm still getting a file not find issue here:
`/Data/private/processed/mwi/ecmwf/seasonal-monthly-individual-members/prate/unrounded-coords/mwi_seasonal-monthly-single-levels_v5_unrounded-coords_2000_1_adm1_stats.csv'`
I looked on the browser version, and there isn't a `unrounded-coords` directory in `prate`

Tinkaa replied:
I am so sorry! Apparently I set two incompatible vars (`use_unrounded_area_coords=True` and `source_cds=False`). Fixed it now..

turnerm replied:
No worries! I'm anyway blocked by my own issues with insync at the moment :zany_face:

turnerm commented on analyses/mwi/dryspells_trigger/exploration/ecmwf_highres_compare_aggr.md:
```diff
@@ -0,0 +1,365 @@
+
+#### Histogram
+Stacked based on occurrence of dry spell    
+Due to so little observations it is not really a distrubtion which makes it hard to compare
```
There are statistical tests you can use if you're interested in knowing whether the data could come from the same underlying distribution (like the KS test)

Tinkaa replied:
Hmm that would maybe have been a good idea to compare the full distribution between methods. However I would say splitting between dry spells and non-dry spells we would have too little data to even estimate a distribution? 

I think for now since we rounded off this exploration I will not make that addition. Is that okay?

turnerm replied:
Of course! Just putting the idea out there.

turnerm commented on analyses/mwi/dryspells_trigger/exploration/ecmwf_highres_compare_aggr.md:
````diff
@@ -0,0 +1,365 @@
+```
+
+```python
+# #select methods for plotting of slides
````
Can these be removed?

Tinkaa replied:
I would keep them since they are used in slide, so to make sure everything is reproducible.

turnerm replied:
Then I would leave then uncommented maybe

turnerm reviewed (approved):
Looks great!

Tinkaa commented on analyses/mwi/dryspells_trigger/09_ecmwf_monthly_skill_dryspells.md:
````diff
@@ -745,7 +752,8 @@ df_pr_ds=compute_miss_false_leadtime(df_ds_for,"dry_spell","for_below_th")
+                                       figsize=(20,8),colp_num=4)
 # fig_cm.savefig(os.path.join(plots_seasonal_dir,f"mwi_plot_formonth_dsobs_cm_lt123456_th{int(threshold_perc)}_perc_{int(probability*100)}_{adm_str}_{month_str}.png"))
 ```
 
````
Aaah yes that is in the part that is not kept up-to-date anymore. Wanted to keep it as reference that we also explored other ideas and mentioned code might break. 
But I guess that exploration is not that useful anymore anyway so will remove it all together
